content/en/docs/instrumentation/go/exporters.md

```diff
@@ -2,56 +2,178 @@
 title: Exporters
 aliases: [/docs/instrumentation/go/exporting_data]
 weight: 50
-cSpell:ignore: otlptrace otlptracehttp
+# prettier-ignore
+cSpell:ignore: otlpmetric otlpmetricgrpc otlpmetrichttp otlptrace otlptracegrpc otlptracehttp promhttp stdouttrace
 ---
 
 {{% docs/instrumentation/exporters-intro go %}}
 
-## OTLP endpoint
+## Console
+
```

Comment:
I recommend some commentary here about when console export is useful (i.e., primarily during development and local debugging)

Reply:
PTAL https://github.com/open-telemetry/opentelemetry.io/pull/3320/commits/a1d044f61ea5c93a6a4e5721a12cf1f3f6b925ae